internal/codespaces/grpc/client_test.go

```diff
@@ -0,0 +1,68 @@
+package grpc
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"testing"
+
+	"github.com/cli/cli/v2/internal/codespaces/grpc/test"
+)
+
```

Comment:
Why not use `Connect`?

It is the public API of the pkg (until `NewClient` was added) yet we do not test it?